react-web/src/components/Session.js

```diff
@@ -1,25 +1,46 @@
 import React from 'react';
-import { PointSelector } from './pointSelector';
+import { Nav, NavItem, TabContent, TabPane, Tabs } from 'patternfly-react';
+import { PointingSession } from './PointingSession';
 
 export class Session extends React.Component {
 
   constructor(props) {
     super(props);
 
-    this.pointSelected = this.pointSelected.bind(this);
-  }
-
-  pointSelected(value) {
-    // Handle point value
+    this.state = {
```

Comment:
I think this can be removed as well once https://github.com/maleck13/storypoint/pull/16 lands too?

Reply:
yep that can be removed once that PR lands